### ISSUE TITLE
Starting Celery 4.0.2 gives a long error message

### ISSUE THREAD

**Issue by alexanderwhatley** [celery/celery#3780]
I'm on Windows 10, and celery is giving me a long error message when I start it. Here is the traceback:

```
c:\users\alexander\anaconda3\lib\site-packages\celery\backends\amqp.py:68: CPendingDeprecationWarning:
    The AMQP result backend is scheduled for deprecation in     version 4.0 and removal in version v5.0.     Please use RPC backend or a persistent backend.

  alternative='Please use RPC backend or a persistent backend.')

 -------------- celery@AlexanderPC v4.0.2 (latentcall)
---- **** -----
--- * ***  * -- Windows-10-10.0.14986-SP0 2017-01-20 21:41:06
-- * - **** ---
- ** ---------- [config]
- ** ---------- .> app:         app:0x23d77ae03c8
- ** ---------- .> transport:   amqp://guest:**@localhost:5672//
- ** ---------- .> results:     amqp://
- *** --- * --- .> concurrency: 4 (prefork)
-- ******* ---- .> task events: OFF (enable -E to monitor tasks in this worker)
--- ***** -----
 -------------- [queues]
                .> celery           exchange=celery(direct) key=celery


[2017-01-20 21:41:06,987: CRITICAL/MainProcess] Unrecoverable error: AttributeError("Can't pickle local object 'Pool.__init__.<locals>.Process'",)
Traceback (most recent call last):
  File "c:\users\alexander\anaconda3\lib\site-packages\celery\worker\worker.py", line 203, in start
    self.blueprint.start(self)
  File "c:\users\alexander\anaconda3\lib\site-packages\celery\bootsteps.py", line 119, in start
    step.start(parent)
  File "c:\users\alexander\anaconda3\lib\site-packages\celery\bootsteps.py", line 370, in start
    return self.obj.start()
  File "c:\users\alexander\anaconda3\lib\site-packages\celery\concurrency\base.py", line 131, in start
    self.on_start()
  File "c:\users\alexander\anaconda3\lib\site-packages\celery\concurrency\prefork.py", line 112, in on_start
    **self.options)
  File "c:\users\alexander\anaconda3\lib\site-packages\billiard\pool.py", line 1008, in __init__
    self._create_worker_process(i)
  File "c:\users\alexander\anaconda3\lib\site-packages\billiard\pool.py", line 1117, in _create_worker_process
    w.start()
  File "c:\users\alexander\anaconda3\lib\site-packages\billiard\process.py", line 122, in start
    self._popen = self._Popen(self)
  File "c:\users\alexander\anaconda3\lib\site-packages\billiard\context.py", line 383, in _Popen
    return Popen(process_obj)
  File "c:\users\alexander\anaconda3\lib\site-packages\billiard\popen_spawn_win32.py", line 79, in __init__
    reduction.dump(process_obj, to_child)
  File "c:\users\alexander\anaconda3\lib\site-packages\billiard\reduction.py", line 99, in dump
    ForkingPickler(file, protocol).dump(obj)
AttributeError: Can't pickle local object 'Pool.__init__.<locals>.Process'

(C:\Users\Alexander\Anaconda3) C:\Users\Alexander\Google Drive\reactfire>Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "c:\users\alexander\anaconda3\lib\site-packages\billiard\spawn.py", line 165, in spawn_main
    exitcode = _main(fd)
  File "c:\users\alexander\anaconda3\lib\site-packages\billiard\spawn.py", line 207, in _main
    self = pickle.load(from_parent)
EOFError: Ran out of input
```

**Comment by alexanderwhatley** [comment]
Just as an update, I included an example of code that causes an issue in this stackoverflow post: http://stackoverflow.com/questions/41775353/celery-throwing-long-error-message

**Comment by AnthonyDeroche** [comment]
It's specified in doc that Celery 4 does not support Windows anymore.

**Comment by georgepsarakis** [comment]
@AnthonyDeroche is correct. Adding also the [documentation link](http://docs.celeryproject.org/en/latest/faq.html?highlight=windows#does-celery-support-windows) for reference.

**Comment by ErfanPY** [comment]
Solution: https://celery.school/celery-on-windows
TLDR:
`celery --app=app.app worker --pool=solo` 
or
`celery --app=app.app worker --pool=threads --concurrency=8`
or
`celery --app=app.app worker --pool=gevent --concurrency=8`
